package.json

```diff
@@ -68,6 +67,7 @@
     "prettier": "^1.13.5",
     "raf": "^3.4.0",
     "raw-loader": "^0.5.1",
+    "react-onclickoutside": "^6.8.0",
```

Comment:
Hey why did we move this to dev-dependencies? For what it looks is used as a dependency.